azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/implementation/LoadBalancerImpl.java

```diff
@@ -225,7 +353,7 @@ public void failure(Throwable t) {
                     public void success(ServiceResponse<Void> result) {
                         callback.success(result);
                         try {
-                            runPostCreationTasks();
+                            afterCreating();
```

Comment:
AfterCreating() updates NIC::ifconfig of VMs to be associated with the LB. Once we have full LB functionality we might want to think about having an async version of AfterCreating to ensure we don't block this `createResourceAsync` call. Similarly for all such internal create or update operations we introduce as a part of LB fluent.